### ISSUE TITLE
Add criticality level to resource type / scope

### ISSUE THREAD

**Issue by roflc0pter** [zalando-stups/essentials#11]
We need a new integer field for scopes (e.g. criticality 1-3).

**Comment by hjacobs** [comment]
I would add a database field similar to `a_criticality_level` in Kio, see https://github.com/zalando-stups/kio/blob/master/resources/db/migration/V6__%20Add%20criticality%20fields.sql#L3

**Comment by hjacobs** [comment]
@kolja we only need to add the criticality level on the `scope` table:

``` sql
ALTER TABLE ze_data.scope
 ADD COLUMN s_criticality_level SMALLINT DEFAULT 2 CHECK (s_criticality_level >= 1 AND
                                                          s_criticality_level <= 3);
```

**Comment by hjacobs** [comment]
And add the `criticality_level` field in the REST API of course :smile:

**Comment by prayerslayer** [comment]
I think this is merged? Can we close?

**Comment by harti2006** [comment]
Fixed by #12
